src/Application.cpp

```diff
@@ -292,6 +313,14 @@ String Application::processStatsPageHTML(const String& var)
 
   return String();
 }
+
+String Application::processScriptFile(const String &var) {
+  if (var == "SENSOR-UPDATE-SECONDS") {
+    return String(AIR_QUALITY_SENSOR_UPDATE_SECONDS);
```

Comment:
I would argue this is too faster in the default case. In the original configuration, that means the page would auto refresh every 2 seconds. While the ESP32 is a pretty decent MCU, Iw Ould want to put that much load on it. I would suggest that the minimum value we use for an auto refresh be 1 minutes.

Reply:
Changed the refresh interval to 60s on the frontend. Currently hardcoded, but might add a widget in the future